core/bin/external_node/src/config/mod.rs

```diff
@@ -531,6 +553,14 @@ impl ExternalNodeConfig {
             .from_env::<OptionalENConfig>()
             .context("could not load external node config")?;
 
+        let api_component_config = envy::prefixed("EN_")
```

Comment:
wdyt about using different prefixes like EN_API and EN_TREE? It would be easier to tell where is certain env variable needed